internal/proto/local/controller/api/resources/accounts/v1/account.proto

```diff
@@ -14,83 +14,83 @@ import "controller/custom_options/v1/options.proto";
+	google.protobuf.Timestamp created_time = 50 [json_name="created_time"];  // @gotags: `class:"public"`
 
 	// Output only. The time this resource was last updated.
-	google.protobuf.Timestamp updated_time = 60 [json_name="updated_time"];
+	google.protobuf.Timestamp updated_time = 60 [json_name="updated_time"];  // @gotags: `class:"public"`
 
 	// Version is used in mutation requests, after the initial creation, to ensure this resource has not changed.
 	// The mutation will fail if the version does not match the latest known good version.
-	uint32 version = 70;
+	uint32 version = 70;  // @gotags: `class:"public"`
 
 	// The type of this Account.
-	string type = 80;
+	string type = 80;  // @gotags: `class:"public"`
 
 	// The ID of the Auth Method that is associated with this Account.
-	string auth_method_id = 90 [json_name="auth_method_id"];
+	string auth_method_id = 90 [json_name="auth_method_id"];  // @gotags: `class:"public"`
 
 	// The attributes that are applicable for the specific Account type.
 	oneof attrs {
-		google.protobuf.Struct attributes = 100 [(custom_options.v1.generate_sdk_option) = true, (custom_options.v1.subtype) = "default"];
+		google.protobuf.Struct attributes = 100 [(custom_options.v1.generate_sdk_option) = true, (custom_options.v1.subtype) = "default"];  // @gotags: `class:"public"`
 		PasswordAccountAttributes password_account_attributes = 101 [
 			(google.api.field_visibility).restriction = "INTERNAL",
 			(custom_options.v1.generate_sdk_option) = true,
 			(custom_options.v1.subtype) = "password"
-		];
+		];  // @gotags: `class:"public"`
 		OidcAccountAttributes oidc_account_attributes = 102 [
 			(google.api.field_visibility).restriction = "INTERNAL",
 			(custom_options.v1.generate_sdk_option) = true,
 			(custom_options.v1.subtype) = "oidc"
-		];
+		];  // @gotags: `class:"public"`
 	}
 
 	// Output only. managed_group_ids indicates IDs of the managed groups that currently contain this account
-	repeated string managed_group_ids = 110 [json_name="managed_group_ids"];
+	repeated string managed_group_ids = 110 [json_name="managed_group_ids"];  // @gotags: `class:"public"`
 
 	// Output only. The available actions on this resource for this user.
-	repeated string authorized_actions = 300 [json_name="authorized_actions"];
+	repeated string authorized_actions = 300 [json_name="authorized_actions"];  // @gotags: `class:"public"`
 }
 
 // Attributes associated only with Accounts with type "password".
 message PasswordAccountAttributes {
 	// The login name of this Account. This is unique per Auth Method.
-	string login_name = 10 [json_name="login_name", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = {this:"attributes.login_name" that: "LoginName"}];
+	string login_name = 10 [json_name="login_name", (custom_options.v1.generate_sdk_option) = true, (custom_options.v1.mask_mapping) = {this:"attributes.login_name" that: "LoginName"}];  // @gotags: `class:"sensitive"`
```

Comment:
Not sure if this should be sensitive, is this user controlled? Could also be secret perhaps?

Reply:
I think this should be sensitive